tests/training_tests/test_twitter_trainer.py

```diff
@@ -0,0 +1,83 @@
+from tests.base_case import ChatBotTestCase
+from unittest import SkipTest
+from mock import Mock, MagicMock
+from chatterbot.trainers import TwitterTrainer
+import os
+import json
+
+
+def get_search_side_effect(*args, **kwargs):
+    from twitter import Status
+
+    current_directory = os.path.dirname(os.path.realpath(__file__))
+    data_file = os.path.join(
+        current_directory,
+        'test_data',
+        'get_search.json'
+    )
+    tweet_data = open(data_file)
+    data = json.loads(tweet_data.read())
+    tweet_data.close()
+
+    return [Status.NewFromJsonDict(x) for x in data.get('statuses')]
+
+
+def get_status_side_effect(*args, **kwargs):
+    from twitter import Status
+
+    current_directory = os.path.dirname(os.path.realpath(__file__))
+    data_file = os.path.join(
+        current_directory,
+        'test_data',
+        'get_search.json'
+    )
+    tweet_data = open(data_file)
+    data = json.loads(tweet_data.read())
+    tweet_data.close()
+
+    return Status.NewFromJsonDict(data.get('statuses')[1])
+
+
+class TwitterTrainerTestCase(ChatBotTestCase):
```

Comment:
The Twitter training test case probably isn't relevant to the ChatterBot corpus data.